README.rst

```diff
@@ -13,6 +13,7 @@ Running commands from the edx-platform directory will default to loading the
 configuration at ``./conf/locale/config.yaml``. You can specify a different
 configuration file with the ``--config`` argument.
 
+* ``i18n_tool detect``
```

Comment:
"Detect" is kind of vague.  Seems like this is "diff", though I'm still not quite sure what two sources are being compared.